app/src/main/java/com/ccc/nameapp/data/source/remote/api/error/BaseException.kt

```diff
@@ -23,7 +23,7 @@ class BaseException : RuntimeException {
         this.errorResponse = response
     }
 
-    fun getMessageError(context: Context? = null): String {
+    fun getMessageError(): String {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/2f989167ae2e76837c8cfc9363f0232fda87311a/app/src/main/java/com/ccc/nameapp/data/source/remote/api/error/BaseException.kt#L26"></span>Function getMessageError has 4 return statements which exceeds the limit of 2.</td>
    </tr>
  </tbody>
</table>